shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -0,0 +1,69 @@
+use cassandra_cpp::*;
+use std::thread::sleep;
+use std::time::Duration;
+use tracing::info;
+
+mod basic_driver_tests;
+
+pub struct CassandraTestContext {
+    pub session: Session,
+}
+
+impl CassandraTestContext {
+    pub fn new() -> Self {
+        CassandraTestContext::new_with_points("127.0.0.1")
+    }
+
+    pub fn new_with_points(contact_points: &str) -> Self {
+        CassandraTestContext::new_with_points_and_port(contact_points, 9042)
+    }
+
+    pub fn new_with_points_and_port(contact_points: &str, port: u16) -> Self {
```

Comment:
Yes, in my comment I did say "it might be worth moving it to be freestanding functions outside of ShotoverManager"

I think it makes sense to keep the test connection creators together, but dont really know what the best way to organize them would be, so for now and possibly permanently we can keep this here.